### PR TITLE
Add documentation for trivial usage of react_component

### PR DESCRIPTION
**Pull request by mrcola91**

### CODE REVIEW

File: a-trivial-procore-react-component.md [hunk 1]
```diff
@@ -1,3 +1,171 @@
...
+};
+
+ExampleComponent.defaultProps = {
+  editable: false,
+};
+
+export default ExampleComponent;
+```
+This is called a "Pure Component".
+
```
Comment (mboperator) [inline a-trivial-procore-react-component.md:82]: Only thing I'd add here is an example of how `import` or `System.import`ing this component looks like in `src/components.js` and how it looks like when we add it to the `export default` block.

**Review by mboperator — COMMENTED** [review]
Fantastic job with this man 👍 

Only one comment, but I feel like addressing it would add a huge amount of value

File: a-trivial-procore-react-component.md (hunk 1 above)
Comment (mrcola91) [reply a-trivial-procore-react-component.md:82]: Addressed

**Review by mboperator — APPROVED** [review]

**Comment by mboperator** [comment]
Merging!

**Pull request merged**